modal/_functions.py

```diff
@@ -970,20 +970,9 @@ def _bind_parameters(
         parent = self
 
         async def _load(param_bound_func: _Function, resolver: Resolver, existing_object_id: Optional[str]):
-            try:
-                identity = f"{parent.info.function_name} class service function"
-            except Exception:
-                # Can't always look up the function name that way, so fall back to generic message
-                identity = "class service function for a parametrized class"
             if not parent.is_hydrated:
-                if parent.app._running_app is None:
-                    reason = ", because the App it is defined on is not running"
-                else:
-                    reason = ""
-                raise ExecutionError(
-                    f"The {identity} has not been hydrated with the metadata it needs to run on Modal{reason}."
-                )
-
+                # While the base Object.hydrate() method appears to be idempotent, it's not always safe
```

Comment:
Should we try to add a unit test that checks the exercises the same path that failed in the integration test?